model/historical_test_data.go

```diff
@@ -302,10 +307,33 @@ func GetHistoricalTestData(ctx context.Context, env cedar.Environment, filter Hi
 	pipeline := filter.queryPipeline()
 	cursor, err := env.GetDB().Collection(historicalTestDataCollection).Aggregate(ctx, pipeline)
 	if err != nil {
-		return nil, errors.Wrap(err, "problem aggregating test data")
+		return nil, errors.Wrap(err, "aggregating test data")
 	}
 	if err = cursor.All(ctx, &data); err != nil {
-		return nil, errors.Wrap(err, "problem unmarshaling aggregated test data")
+		return nil, errors.Wrap(err, "unmarshaling aggregated test data")
+	}
+
+	// TODO (16140): Remove this comparison logic once we do the BSON to
+	// Parquet cutover.
+	var prestoData []AggregatedHistoricalTestData
+	cursor, err = env.GetDB().Collection(prestoHistoricalTestDataCollection).Aggregate(ctx, pipeline)
```

Comment:
How does the data get into the `prestoHistoricalTestDataCollection` in the first place? Does it come from the MARS job?

Reply:
Yes! Ryan and I are going to write the MARS job Monday to do the nightly ETL of data.